src/main/java/com/alexlasota/medical_clinic_proxy/client/MedicalClinicFallback.java

```diff
@@ -0,0 +1,32 @@
+package com.alexlasota.medical_clinic_proxy.client;
+
+import com.alexlasota.medical_clinic_proxy.model.VisitDto;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+
+@Component
+public class MedicalClinicFallback implements MedicalClinicClient {
+
+    @Override
+    public List<VisitDto> getVisitsByPatientId(Long patientId) {
+        return Collections.emptyList();
```

Comment:
Moim zdaniem taki fallback nie jest pomocny.
Wyobraz sobie ze jako uzytkownik chcesz zobaczyc swoje wizyty a nagle ci zadnych nie pokazuje. To jest sytuacja gdzie moim zdaniem powinien zostac rzucony wyjatek i uzytkownik powinien dostac odpowiedni komunikat np:

"We are currently unable to show you your visit. Please try again later "